src/tests/FunctionalTests/iOS/Simulator/AOT/iOS.Simulator.Aot.Test.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We could try something like

```xml
<Project Sdk="Microsoft.NET.Sdk" TreatAsLocalProperty="MonoForceInterpreter" >
```

and then set `MonoForceInterpreter` to false in this project.

Reply:
I'll try that in a follow up PR, one to just target running this particular test project on CI.